ctf/tests/core_spec.lua

```diff
@@ -0,0 +1,112 @@
+function assert_tables_equal(expected, actual)
+    for key,value in pairs(expected) do
+        assert.equal(expected[key], actual[key])
+    end
+end
+
+function assert_keysets_are_equal(sample, actual)
+    for key in pairs(sample) do
+        assert.not_nil(actual[key])
+    end
+    for key in pairs(actual) do
+        assert.not_nil(sample[key])
+    end
+end
+
+
+describe("tests orderPairs", function()
+    setup(function()
+        require("ctf/tests/mock/minetest")
```

Comment:
How do you find the path that the script is located at?
If you run busted in the root of the project,  the 'relative path' would start at the root instead of the mod.